### ISSUE TITLE
i18n: E-mail-templates not translated

### ISSUE THREAD

**Issue by Af0x** [osTicket/osTicket#5615]
Hello,

uploading a language pack (i.e. German) before installation and selecting the newly added language as default during installation will not translate the e-mail-templates. 

Working with 1.14.2. Downloaded the language file few minutes ago.

Regards,

Af0x

**Comment by JediKev** [comment]
@Af0x 

This was fixed with:
- https://github.com/osTicket/osTicket/pull/5550

With this being said, you will have to wait until we build the new language packs as this is an issue with the PHAR's structure.

Cheers.

**Comment by Af0x** [comment]
Just to clarify: Even if I install with 1.14.3 and the most recent language packs this issue will persist right now?

**Comment by JediKev** [comment]
@Af0x 

Yes, the issue is the way the language packs were built, so the issue is with the language packs themselves not the codebase. I will update this thread once the language packs are rebuilt.

Cheers.

**Comment by Af0x** [comment]
Thank you. I don´t want to stress but do you know when this will be? I ask because I am in the midst of a reinstall and consider a restart if this will happen in a very timely manner.

**Comment by chonki** [comment]
Are there any updates or are there manual install instructions?
